pyrogram/methods/chats/get_chat_onlines.py

```diff
@@ -0,0 +1,59 @@
+#  Pyrogram - Telegram MTProto API Client Library for Python
+#  Copyright (C) 2017-2021 Dan <https://github.com/delivrance>
+#
+#  This file is part of Pyrogram.
+#
+#  Pyrogram is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU Lesser General Public License as published
+#  by the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  Pyrogram is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public License
+#  along with Pyrogram.  If not, see <http://www.gnu.org/licenses/>.
+
+from typing import Optional, Union
+
+from pyrogram import raw
+from pyrogram.scaffold import Scaffold
+
+
+class GetChatOnlines(Scaffold):
+    async def get_chat_onlines(
+        self,
+        chat_id: Union[int, str]
+    ) -> Optional[int]:
```

Comment:
The function does not return None. Just int or raise an exception.

```suggestion
    ) -> int:
```

Reply:
😳